javascript/javascript2/week1/spiritAnimals.js

```diff
@@ -0,0 +1,45 @@
+const button1 = document.querySelector('#button1')
+const button2 = document.querySelector('#button2')
+
+button1.addEventListener('click', onClick)
+button2.addEventListener('mouseover', mouseOver)
```

Comment:
Almost all of the login in `onClick` and `mouseOver` are the same, so it's a bit redundant to have two methods doing the same thing. I would suggest doing something like this:
```javascript
function findSpiritAnimal(inputNameId, outputId) {
   const index = Math.floor(Math.random() * 10);
   const inputName = document.getElementById(inputNameId).value;

   if(!inputName){
      document.getElementById(outputId).innerText = 'please enter a name';
   } else {
      document.getElementById(outputId).innerText =  inputName + ' - ' + spiritAnimals[index] ;
   }
}
```